coordinator/coordinator_test.go

```diff
@@ -148,10 +149,14 @@ func TestStop(t *testing.T) {
 	require.Equal(t, Running, c.status.State)
 	c.mut.RUnlock()
 
+	time.Sleep(1 * time.Millisecond)
```

Comment:
Why using `1 * `?

Reply:
No reason, force of habit. I'll fix that :)